lib/si-data-pg/src/lib.rs

```diff
@@ -193,6 +199,9 @@ impl Default for PgPoolConfig {
             pool_timeout_wait_secs: None,
             pool_timeout_create_secs: None,
             pool_timeout_recycle_secs: None,
+            pool_total_connection_lifetime_secs: 20 * 3600, // 20 hours (RDS Proxy has a 24-hour max)
+            pool_idle_connection_lifetime_secs: 6 * 3600,   // 6 hours
+            pool_lifetime_check_interval_secs: None,
```

Comment:
Yeah, those all seem reasonable, nice that you noted the reason for the total lifetime is because of RDS Proxy's defaults.